include/range/v3/range/access.hpp

```diff
@@ -726,6 +726,9 @@ namespace ranges
 
         using ranges::iterator_t;
         using ranges::sentinel_t;
+
+        template<typename T>
+        RANGES_INLINE_VAR constexpr bool enable_borrowed_range = ranges::enable_safe_range<T>;
```

Comment:
I would rather make `enable_borrowed_range` the primary, and mark `enable_safe_range` as deprecated. Ditto for the other entities that got renamed.